app/components/ProductList/index.js

```diff
@@ -37,9 +40,10 @@ class ProductList extends React.Component {
   }
 
   orderItems (field)  {
-      return  products.slice().sort(this.compareStrings(this.state.sortBy));
+      return  this.state.products.slice().sort(this.compareStrings(this.state.sortBy));
```

Comment:
- Just use the products from the props to sort them
- this.compareStrings should be renamed to createComparer, because its not actually comparing anything but returning a compare function according to sortBy parameter. But is nice to get your compare function this way